influxql/ast_test.go

```diff
@@ -295,6 +295,7 @@ func TestSelectStatement_HasWildcard(t *testing.T) {
 
 	for i, tt := range tests {
 		// Parse statement.
+		t.Logf("statement: %s", tt.stmt)
```

Comment:
I can add the indexes, but those are typically the useless part for debugging, as counting down are pretty hard.  Adding this line was what enabled me to easily search for and find the offending query.  I believe our opinions differ :-).